lib/srv/db/common/test.go

```diff
@@ -92,6 +92,17 @@ func (cfg *TestServerConfig) Port() (string, error) {
 	return port, nil
 }
 
+// AuthClientGenerateCert contains the required methods to Generate mTLS certificate to be used
+// by the postgres TestServer.
+type AuthClientGenerateCert interface {
```

Comment:
Nit: Minor, but I think `AuthClientCA` would be a bit more apt name for this interface.